sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/OrchestrationController.java

```diff
@@ -30,12 +32,11 @@
 @RestController
 @RequestMapping("/orchestration")
 class OrchestrationController {
-  static final LLMModuleConfig LLM_CONFIG =
-      LLMModuleConfig.create().modelName("gpt-35-turbo").modelParams(Map.of());
+  LLMModuleConfig llmConfig = GPT_35_TURBO;
 
   private final OrchestrationClient client = new OrchestrationClient();
   private final OrchestrationModuleConfig config =
-      new OrchestrationModuleConfig().withLlmConfig(LLM_CONFIG);
+      new OrchestrationModuleConfig().withLlmConfig(llmConfig);
```

Comment:
(Minor)
We could just omit the field declaration altogether, right?

Reply:
Nope, used in testing

Reply:
Removed